ulid_test.go

```diff
@@ -258,6 +258,12 @@ func TestParseRobustness(t *testing.T) {
 			}
 		}()
 
+		// quick.Check doesn't constrain input,
+		// so we need to do so artificially.
+		if s[0] > '7' {
+			s[0] = '7'
```

Comment:
I’d sum the previous value with a modulo so that it doesn’t go over ‘7’. That way the distribution would be a bit more spread out, I think.